.github/workflows/ci.yaml

```diff
@@ -0,0 +1,89 @@
+name: "CI"
+
+concurrency:
+  group: "ci-${{ github.head_ref || github.run_id }}"
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  merge_group:
+  push:
+    branches: [main]
+
+jobs:
+  ci:
+    name: "Execute (PHP ${{ matrix.php-version }} with ${{ matrix.dependencies }} deps)"
+    runs-on: "ubuntu-22.04"
+    container:
+      image: "${{ matrix.php-image }}"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        dependencies:
+          - "lowest"
+          - "locked"
+          - "highest"
+        php-version:
+          - "8.1"
+          - "8.2"
+          - "8.3"
+        include:
+          - php-version: "8.1"
+            php-image: "public.ecr.aws/docker/library/php:8.1-cli-alpine"
+          - php-version: "8.2"
+            php-image: "public.ecr.aws/docker/library/php:8.2-cli-alpine"
+          - php-version: "8.3"
+            php-image: "public.ecr.aws/docker/library/php:8.3-cli-alpine"
+
+    steps:
+      - name: "Install OS dependencies"
+        run: apk add --no-cache bash git su-exec
+
+      - name: "Install composer"
+        run: curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin --filename=composer --version=2.6.5
```

Comment:
```suggestion
        run: curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/bin --filename=composer --version=2.7.1
```

Reply:
I was just going to let Renovate pick this up immediately, but why not!